Patch 9.17 Update

jtborn opened the pull request:
Hi, I forked your repo and added the latest changes:
Added Hextech origin and champions
Added Pantheon to Dragon/Guardian
Updated items and class origins with the latest patch changes to the ItemStrings_en-US.xaml descriptions.
Code builds, adjusted some of the row sizes so that the text would all fit.

jtborn committed:
Patch 9.17 Update

Added Hextech champions and Pantheon, updated item descriptions to recent patch. Updated classes and origins with latest changes. Some general cleanup.

jtborn committed:
Update TFT Overlay.csproj

Reverted system-specific path includes

jtborn committed:
Update TFT Overlay.csproj

jtborn commented:
One note, I did not update the version #, so you'll have to update that to whatever number you want for the auto-updater to work.

sunuanxiaopy commented:
Hi, I think just2good is not updated, can you convert it into. Exe for us non-programmer players? Thank you very much. Look forward to your next update.

sunuanxiaopy commented:
Hi, I think just2good is not updated, can you convert it into. Convert to.EXE format ? Thank you very much. Look forward to your next update.

jtborn commented:
Hi sunuanxiaopy, I was hoping J2G would just merge in my updates but it doesn't look like they're going to so when I get home from work tonight I will upload the .exe to my repo and change my readme file to point to it and comment again here so you get a notification.

sunuanxiaopy commented:
thank u so much❤️

> 在 2019年9月9日，23:24，jtborn <notifications@github.com> 写道：
> 
> Hi sunuanxiaopy, I was hoping J2G would just merge in my updates but it doesn't look like they're going to so when I get home from work tonight I will upload the .exe to my repo and change my readme file to point to it and comment again here so you get a notification.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.

jtborn committed:
Updated Version

jtborn committed:
Update Readme

jtborn commented:
I've published my release here:
https://github.com/jtborn/TFT-Overlay/releases/tag/V2.0.6
It works the same way. When the new patch drops I will update the relevant items and update the version number so the auto-updater feature will still work and it'll pull the newest update for you automatically. Let me know if you have any issues! Thanks.

sunuanxiaopy commented:
That’s so sweet and convenient，thank u

> 在 2019年9月10日，14:42，jtborn <notifications@github.com> 写道：
> 
> It works the same way. When the new patch drops I will update the relevant items and update the version number so the auto-updater feature will still work and it'll pull the newest update for you automatically.
>

Tobu76 commented:
Hi,jtborn.Could you publish the latest 9.18 version update?You know ,there are plenty of new items .Looking for your update and thank u very much!

jtborn committed:
Patch 9.18 Update

jtborn commented:
Hey Tobu76, sorry I've been at work. I pushed 9.18 to my branch. It's V2.0.8 let me know if the auto-updater works. I added Neeko's Help (I'll crop it into a circle later, just didn't have the template that J2G was using so I have to create my own in photoshop) and the patch changes but the Sparring Gloves items don't drop until 9.19 right? I haven't played yet today just going off patch notes.

jtborn committed:
CN localization fix

golol committed:
Update ItemStrings_zh-CN.xaml

jtborn committed:
Auto Updater Fix

Deleted redundant Version file, made Neeko's Help gold border match, fixed Auto Updater to point to my repo.

jtborn committed:
Merge pull request #1 from golol/master

Update ItemStrings_zh-CN.xaml

jtborn committed:
Update Version.cs

Pushing out new Ver with CN localization fixes

jtborn committed:
Merge branch 'master' of https://github.com/jtborn/TFT-Overlay

jtborn committed:
Update README.md

jtborn committed:
New Items

Initial commit of new items
Updated Item Builder columns to match main tab

jtborn committed:
Updated Readme and Localization

jtborn committed:
Update in preparation for Patch 9.19

Fixed broken zh-CN from pull request, Added all new items to Item Builder and Main panel

jtborn committed:
Toltip Update

Usability Update for Tooltip position

jtborn committed:
Pushing new version

jtborn committed:
Update App.xaml.cs

jtborn committed:
Update to CN localization

jtborn committed:
Update ItemStrings_zh-CN.xaml

jtborn committed:
Update Version.cs

jtborn committed:
Update README.md

jingfan2646 commented:
Hello, I am a Chinese players, help you to translate the new Chinese patch

jtborn commented:
Hi jingfan, please do. I've added the new items in English to the zh-CN.xaml localization file to make it easier for you to translate directly from that file without having to merge in my US.xaml code file.

jingfan2646 commented:
Here have a new Chinese translation, please download
https://github.com/jingfan2646/TFT-Overlay/tree/master/Resource/Localization

jtborn commented:
Thanks, I will be able to merge it later tonight when I get home.

jtborn committed:
CN update

jtborn committed:
Update README.md

jingfan2646 commented:
Hello, I found that I had a little translation an item has been added on now, if it's convenient please help patches, thank you

jingfan2646 commented:
Hello, I found several translation errors, has now been repaired, I hope you can repair

jtborn commented:
Hi, I'll be able to fix it when I'm home from work in about 8 hours. If you can submit a pull request to my branch it makes it easier to merge for future fixes.

Edit: I've now pushed the release with your latest fixes. I didn't merge in the Yordle 9 buff as that is still unconfirmed (I'll wait for the official 9.19 Patch Notes) and it looks like you changed Warmogs to 8% but looking at the Garena patch notes it looks like it should be 6%. Let me know if you think that's wrong. I know it's 6% on US servers.

jtborn committed:
CN translation updates

jtborn committed:
Patch 9.19 Update

Updated items to riot's new names.
Added Patch 9.19 changes.
Updated the CN items I could

jtborn committed:
Update README.md

jtborn committed:
Update README.md

jtborn committed:
Update README.md

Alima777 committed:
some update of items in detail

jtborn committed:
Merge pull request #4 from Alima777/master

Update ItemStrings_zh-CN.xaml in detail

jtborn committed:
Update to US Patch 9.20

jtborn committed:
Update README.md

jtborn committed:
ItemsTextless bug fix for BF + Gloves

When clicking the BF sword, it was showing the old recipe of BF + BF = Infinity Edge instead of BF + Gloves

jtborn committed:
Patch 9.21 changes

jtborn committed:
Initial push for Set 2

Some initial origins for set 2, also began cleanup of old xml for classes/origins. The split-up of text for the sake of color-coding words doesn't seem worth the hassle especially with the roster now growing.

jtborn committed:
Element Updates

Element descriptions, making all descriptions 2 lines max.

jtborn committed:
remove assassin.jpg blademaster.jpg brawler.jpg elementalist.jpg guardian.jpg gunslinger.jpg hextech.png knight.jpg ranger.jpg shapeshifter.jpg sorcerer.jpg

jtborn committed:
remove demon.jpg glacial.jpg imperial.jpg phantom.jpg void.jpg dragon.jpg ninja.jpg pirate.jpg wild.jpg exile.jpg hextech.jpg noble.jpg robot.jpg yordle.jpg

jtborn committed:
Class Origins updates

Added some more champion icons, waiting on others when the base icon doesn't match the skin used for TFT.

jtborn committed:
remove demon.jpg glacial.jpg imperial.jpg phantom.jpg void.jpg dragon.jpg ninja.jpg pirate.jpg wild.jpg exile.jpg hextech.jpg noble.jpg robot.jpg yordle.jpg

jtborn committed:
remove assassin.jpg blademaster.jpg brawler.jpg elementalist.jpg guardian.jpg gunslinger.jpg hextech.png knight.jpg ranger.jpg shapeshifter.jpg sorcerer.jpg

jtborn committed:
More updates for set 2

jtborn committed:
Set 2 Finishes

jtborn committed:
Update README.md

jtborn committed:
Update Version.cs

jtborn committed:
More Set 2 Updates

Updated item values, synergy stat values, CN lang updates, changed Cursed Blade recipe to Runaan's Hurricane, adjusted player damage formula and re-roll % chance stats

jtborn committed:
CN lang update

SuTree committed:
Update ItemStrings_zh-CN.xaml

Fix Item name by China client.

jtborn committed:
Merge pull request #13 from SuTree/master

Update ItemStrings_zh-CN.xaml

jtborn committed:
Updates

Updated CN items to client names
Added missing origin/class pictures
Removed "variable" origin and added Qiyana and Lux to each element that they can spawn as for more clarity

jtborn committed:
Corrected Unit Costs

jtborn committed:
Patch 9.23

jtborn committed:
Patch 10.1

See readme

jtborn committed:
CN translations

Rumour999 committed:
Update and rename Resource/Localization/ItemStrings_zh-CN.xaml to 资源/本土化/ItemStrings_zh-CN.xaml

Checked the text being used, Some text hasn't been updated for a long time.

jtborn committed:
Merge pull request #18 from TheRMM/patch-1

Update  Resource/Localization/ItemStrings_zh-CN.xaml

jtborn committed:
Update to Patch 10.2

jtborn committed:
Update README.md

jtborn committed:
CN lang updates

jtborn committed:
Deleted extra folder

Rumour999 committed:
更新国服10.2 & 10.3

jtborn committed:
Merge pull request #19 from TheRMM/master

update China mainland version 10.2 & 10.3

jtborn committed:
Patch 10.3

jtborn committed:
Update version

Rumour999 committed:
更新国服10.4

Rumour999 committed:
更新10.5

jtborn committed:
Merge pull request #20 from TheRMM/master

update China mainland version 10.5

Commit:
New release for CN updates Patch 10.5

Commit:
Set 3 Initial Checkin

Added origins/classes to US file, added/updated champ images

Commit:
Set 3 Items

jtborn committed:
Set 3 Ready to go

color97 committed:
china mainland updates 10.6 patch
update china mainland 10.6 patch Item localization
update china mainland 10.6 patch origins+classes localization

color97 committed:
update china mainland 10.6 patch, item updates & balance

jtborn committed:
Merge pull request #22 from color97/master

china mainland updates 10.6 patch & localization

jtborn committed:
CN updates
